assignments/arrays.js

```diff
@@ -63,19 +63,35 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
+// for (let i = 0; i < inventory.length; i++) {
+//     if (inventory[i].id === 33) {
+//         console.log(inventory[i].car_make);
+//         console.log(inventory[i].car_model);
+//         console.log(inventory[i].car_year);
+//     }
+// }
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// let lastCar = 0;
+// for (let i = 0; i < inventory.length; i++) {
+//     if (inventory[i].id === inventory[inventory.length-1].id) {
+//         console.log(inventory[i].car_make);
+//         console.log(inventory[i].car_model);
+//     }
+// }
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+var unsorted = [];
+var carModels = [];
+for (let i = 0; i < inventory.length; i++) {
+    unsorted = inventory[i].car_model;
+    // unsorted[i].sort;
+    console.log(unsorted);
+    // console.log(unsorted);
+}
+
```

Comment:
Adding a console.log in this line to list your unsorted array is better than logging it every time 'i' is incremented. This way your console does not get cluttered. 
Alternatively using the array.sort method might be useful to sort said array.